packages/p2p-v2/src/pages/buy-sell/screens/BuySellTable/BuySellTable.tsx

```diff
@@ -44,6 +60,13 @@ const BuySellTable = () => {
                     tableClassname=''
                 />
             )}
+            <RadioGroupFilterModal
+                isModalOpen={isFilterModalOpen}
+                list={SORT_BY_LIST}
+                onRequestClose={() => setIsFilterModalOpen(false)}
+                onToggle={onToggle}
+                selected={sortDropdownValue as string}
```

Comment:
@ameerul-deriv, just a question, what is the selected value on the UI if  `sortDropdownValue` is `undefined`?

```suggestion
                selected={sortDropdownValue as string}
```

Reply:
it shouldn't be left as undefined because theres only two options for the user, but in BE they have the option for undefined, but we never pass it, so this is just to fix the issue for undefined

Reply:
but it shouldn't ever be undefined

Reply:
if it's `undefined`, there is a default option I assume right?

Reply:
yup its always rate thats default, but I think the TSortByList is affecting the typing

Reply:
Alright, I was asking, because we were asserting it to different types in a few places, so I thought type narrowing would be best used here. I can't clearly see it now, but I think something can be done. LGTM none the less

Reply:
for sure, i will see if i can change it in the other cards coming